### PR TITLE
AlarmPeriodMinuted should be not greater than 1 day (Fixing < to <=)

### PR DESCRIPTION
**Pull request by dashaaa**
### Background

We check that Alarm is not longer than one day. The backend includes precisely one day into acceptable value. Fixing PAT tests to comply.

### Changes

- AlarmPeriodMinutes <= 1440 (instead of <)

### CODE REVIEW

File: tests/unit/panther_analysis_tool/Pipfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
```
Comment (maxrichie5) [inline tests/unit/panther_analysis_tool/Pipfile:11]: Why do the unit tests need a pip file?

File: broken.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+AnalysisType: lookup_table
```
Comment (maxrichie5) [inline broken.yml:1]: Can we delete this file? 😄

**Review by maxrichie5 — COMMENTED** [review]
Awesome! Couple comments.

Make sure to upgrade the PAT version! 😊

File: tests/unit/panther_analysis_tool/Pipfile (hunk 1 above)
Comment (dashaaa) [reply tests/unit/panther_analysis_tool/Pipfile]: Sorry, the random file that "git add ." added to my commit. Deleted

File: broken.yml (hunk 2 above)
Comment (dashaaa) [reply broken.yml]: Sure, my mistake, sorry

**Review by maxrichie5 — COMMENTED** [review]
Just a minor version bump and this looks good to go!

**Comment by cdzombak** [comment]
I'm going to go ahead and merge this @maxrichie5, ahead of your https://github.com/panther-labs/panther_analysis_tool/pull/251 which includes the version # bump.

@dashaaa this change will go out in 0.19.5 in the near future. Thank you!

**Pull request merged**